src/DateRangePicker.jsx

```diff
@@ -6,12 +6,12 @@ import calendar from 'calendar';
 
 import BemMixin from './utils/BemMixin';
 import CustomPropTypes from './utils/CustomPropTypes';
-import Legend from './Legend';
+import Legend from './Legend.jsx';
```

Comment:
I needed to add those to get coverage with isparta working. To be honest, I have not figured out why. Happy to remove both the extensions and coverage support.